backend/test/modules/doubles-tournament/core/application/CallingEnqueuedMatches.spec.ts

```diff
@@ -0,0 +1,160 @@
+import { testDoublesTournamentsModule } from './TestDoublesTournamentsModule';
+import { FromListIdGeneratorStub } from '../../../../test-support/shared/core/FromListIdGeneratorStub';
+import { MatchWasQueued } from '../../../../../src/modules/doubles-tournament/core/domain/event/MatchWasQueued';
+import { InMemoryCommandBus } from '../../../../../src/shared/infrastructure/core/application/command/InMemoryCommandBus';
+import { TournamentTableWasBooked } from '../../../../../src/modules/tournament-tables/core/domain/event/TournamentTableWasBooked';
+import { TournamentTableWasReleased } from '../../../../../src/modules/tournament-tables/core/domain/event/TournamentTableWasReleased';
+import { CallMatch } from '../../../../../src/modules/doubles-tournament/core/application/command/CallMatch';
+import { MatchWasCalled } from '../../../../../src/modules/doubles-tournament/core/domain/event/MatchWasCalled';
+
+describe('Calling Enqueued Matches', () => {
+  const team1Id = 'Team1Id';
+  const team2Id = 'Team2Id';
+  const team3Id = 'Team3Id';
+  const team4Id = 'Team4Id';
+  const tournamentId = 'TournamentId';
+  const commandBus = new InMemoryCommandBus();
+  const spy = jest.spyOn(commandBus, 'execute');
+
+  it('When matches were enqueued and only one table was released then call the match with lower matchNumber', async () => {
+    //Given
+    const currentTime = new Date();
+    const entityIdGen = FromListIdGeneratorStub([team1Id, team2Id, team3Id, team4Id]);
+    const doublesTournament = testDoublesTournamentsModule(currentTime, entityIdGen);
```

Comment:
No tego commandBus co robisz wyżej, gdzieś musisz przekazac do modułu. To jest problem tutaj.
Bo jak widzę sobie tworzysz obiekt, w 16 i spy na nim na 17. A nigdzie nie mówisz, że to właśnie ten obiekt ma być wykorzystywany przez moduł :)